uportal-war/src/main/java/org/apereo/portal/security/IPermission.java

```diff
@@ -112,42 +112,59 @@
     /*
      * PortletMode permissions
      */
-    public static final String PORTLET_MODE_CONFIG = "PORTLET_MODE_CONFIG";
+    String PORTLET_MODE_CONFIG = "PORTLET_MODE_CONFIG";
 
     /*
      * UP_GROUP (GaP) Permissions
      */
 
-    public static final String VIEW_GROUP_ACTIVITY = "VIEW_GROUP";
-    public static final String CREATE_GROUP_ACTIVITY = "CREATE_GROUP";
-    public static final String DELETE_GROUP_ACTIVITY = "DELETE_GROUP";
-    public static final String EDIT_GROUP_ACTIVITY = "EDIT_GROUP";
+    String VIEW_GROUP_ACTIVITY = "VIEW_GROUP";
+    String CREATE_GROUP_ACTIVITY = "CREATE_GROUP";
+    String DELETE_GROUP_ACTIVITY = "DELETE_GROUP";
+    String EDIT_GROUP_ACTIVITY = "EDIT_GROUP";
 
     /** Activity string for adding a tab to your personal layout */
-    public static final String ADD_TAB_ACTIVITY = "ADD_TAB";
+    String ADD_TAB_ACTIVITY = "ADD_TAB";
 
     /** Non-owner-specific view activity (used by ERROR_PORTLET) */
-    public static final String VIEW_ACTIVITY = "VIEW";
+    String VIEW_ACTIVITY = "VIEW";
 
-    public static final String VIEW_USER_ACTIVITY = "VIEW_USER";
-    public static final String VIEW_USER_ATTRIBUTE_ACTIVITY = "VIEW_USER_ATTRIBUTE";
-    public static final String IMPERSONATE_USER_ACTIVITY = "IMPERSONATE";
+    /**
+     * Determines whether a user is visible within the portal.
+     */
+    String VIEW_USER_ACTIVITY = "VIEW_USER";
+
+    /**
+     * Governs which user attributes are visible within the portal, applies
+     * to attributes of others as well as one's own attributes.
+     */
+    String VIEW_USER_ATTRIBUTE_ACTIVITY = "VIEW_USER_ATTRIBUTE";
+
+    /**
+     * Governs the visibility of one's own user attributes.
```

Comment:
Opportunity here to clarify that the `VIEW_USER_ATTRIBUTE` activity is a superset of this activity.